app/views/devise/sessions/_new.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class="login-box login-page mx-auto w-100" >
+  <div class="login-logo">
+    <%= image_tag ("CubreJeta.svg"), :style => "width: 45px" %>
+    <a href="../../index2.html"><b>Covid</b>App</a>
+  </div>
+  <!-- /.login-logo -->
+  <div class="card">
+    <div class="card-body login-card-body">
+      <p class="login-box-msg">Sign in to start your session</p>
+
+      <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+        <div class="input-group mb-3">
+          <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control" %>
+          <div class="input-group-append">
+            <div class="input-group-text">
+              <span class="fas fa-envelope"></span>
+            </div>
+          </div>
+        </div>
+        <div class="input-group mb-3">
+          <%= f.password_field :password, autocomplete: "current-password", class:"form-control" %>
+          
+          <div class="input-group-append">
+            <div class="input-group-text">
+              <span class="fas fa-lock"></span>
+            </div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-8">
+            <div class="icheck-primary">
+              <%= f.check_box :remember_me %>
+              <label for="remember">
+                Remember Me
+              </label>
+            </div>
+          </div>
+          <!-- /.col -->
+          <div class="col-4">
+            <%= f.submit "Log in", class: "btn btn-primary btn-block" %>
+          </div>
+          <!-- /.col -->
+        </div>
```

Comment:
In my opinion there is a lot of divs, try to use more tags in your html for better structure.